.github/workflows/validate_install_plans.yml

```
@@ -0,0 +1,28 @@
+name: Validate install plans
+
+on: [workflow_dispatch, pull_request]
```

I think this is what we need to do to ensure that this runs with the secrets / version of the workflow that we have in the base branch.

```suggestion
on: [workflow_dispatch, pull_request_target]
```

---

im not 100% sure of why things work atm 🙈 

but i dont think we need this. as an example, this workflow (or the previous iteration) didn't have pull_request_target, but it has been running like that all this time. i just tested it out with a little change on a fork i made, and it works there.

i think this running & working comes from our actions approval setting underneath `Fork pull request workflows from outside collaborators`. So my theory is because we have a setting checked there, we can use pull_request instead of pull_request_target for collaborators we approve.